README.md

```diff
@@ -31,15 +33,17 @@ This repository contains some helper items for working with the graphics-core22
        stage-packages:
      # ...
 
-     graphics-core22:
+     graphics-coreXX:
        after: [my-app]
-       source: https://github.com/MirServer/graphics-core22.git
+       source: https://github.com/MirServer/graphics-coreXX.git
```

Comment:
This source is wrong, both now and in the future described in the PR:

> This change makes this repository support more than just -core22. Because all the artifacts are versioned anyway, this makes perfect sense. It should be renamed to graphics-core, subsequently.